base/bucket_gocb_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"testing"
 
 	"github.com/couchbase/gocb"
-	"github.com/couchbase/sg-bucket"
+	sgbucket "github.com/couchbase/sg-bucket"
```

Comment:
Should manually remove any noise like this from PRs (and ideally get IDE settings aligned to avoid this kind of thing.

Reply:
Okay I can manually remove. Need to figure out what is different. Ben and I checked and it looks like ours are the same.

Reply:
My IDE isn't adding sgbucket when I save this file, we might need to take another look through preferences to compare.

Reply:
Ah ok, apologies. I thought we had.